taxi/models.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return f"{self.name} {self.country}"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=63, unique=True)
+
+    class Meta:
+        ordering = ('username', )
+
+    def __str__(self) -> str:
+        return f"{self.username} ({self.first_name} {self.last_name})"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+        related_name='cars'
+    )
+    drivers = models.ManyToManyField(
+        settings.AUTH_USER_MODEL,
+        related_name='cars'
+    )
+
+    def __str__(self) -> str:
+        return f"{self.model} {self.manufacturer}"

```

Comment:
The `__str__` method for the `Car` class is currently returning the `manufacturer` object, which may not be a string. It would be better to return a specific attribute of the `manufacturer`, such as its name.